### PR TITLE
Add GitHub issue templates

### PR DESCRIPTION
**Pull request by necolas**
Adds issue and PR templates based on those we developed for React Native. Issues that are questions or related to documentation will automatically be labelled as "Resolution: Invalid" to help with triaging.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,14 +1,4 @@
-<!--
-  Note: if the issue is about documentation or the website, please file it at:
-  https://github.com/reactjs/reactjs.org/issues/new
--->
+👉 Please follow one of these issue templates:
+- https://github.com/facebook/react-native/issues/new/choose
```
Comment (gaearon) [inline .github/ISSUE_TEMPLATE.md]: This says React Native?

**Comment by gaearon** [comment]
What about feature requests? Do we consider these questions?

**Comment by necolas** [comment]
Where is the line between "RFC" and "Feature Request"?

**Comment by shivamvora** [comment]
exactly What about feature requests? Do we consider these questions?

**Pull request merged**